packages/dropdowns/src/Menu/Items/Item.spec.js

```diff
@@ -0,0 +1,205 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { render, fireEvent } from 'garden-test-utils';
+import { Dropdown, Trigger, Menu, Item } from '../../';
+
+describe('Item', () => {
+  it('throws error if value is not provided', () => {
+    console.error = jest.fn(); // eslint-disable-line no-console
+    const Example = () => (
+      <Dropdown>
+        <Trigger>
+          <button>Test</button>
+        </Trigger>
+        <Menu>
+          <Item>Item 1</Item>
+        </Menu>
+      </Dropdown>
+    );
+
+    expect(() => {
+      render(<Example />);
+    }).toThrow('All Item components require a `value` prop');
+  });
+
+  it('does not throw if value is not provided while item is disabled', () => {
+    console.error = jest.fn(); // eslint-disable-line no-console
```

Comment:
This is affecting other tests too, right? Maybe a `beforeEach` + `afterEach` should be used so that any new tests written won't unknowingly rely on this?

Reply:
I've updated all `console.error` tests including those in other packages. These are now mocked correctly.